pkgs/applications/editors/vim/plugins/overrides.nix

```diff
@@ -846,6 +846,16 @@ self: super: {
     dependencies = with self; [ vim-maktaba ];
   });
 
+  # Due to case-sensitivety issues, the hash differs on Darwin systems, see:
+  # https://github.com/NixOS/nixpkgs/issues/157609
+  vim-colorschemes = super.vim-colorschemes.overrideAttrs (old: {
+    src = old.src.overrideAttrs (srcOld: {
+      postFetch = srcOld.postFetch + lib.optionalString (!stdenv.isDarwin) ''
```

Comment:
```suggestion
      postFetch = (srcOld.postFetch or "") + lib.optionalString (!stdenv.isDarwin) ''
```

Reply:
The `or ""` is only added to prevent evaluation errors if the overwriten Derivation for some reason no longer contain postFetch.

Reply:
That's what I assumed yeah.